mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -22,10 +22,16 @@ import { getUUID } from '../../common/utils/ActionUtils';
 export const CHART_TYPE_LINE = 'line';
 export const CHART_TYPE_BAR = 'bar';
 
+// Polling interval
+export const METRICS_PLOT_POLLING_INTERVAL_MS = 5000;
```

Comment:
Can we increase this to 10 seconds?

Reply:
In general, what happens if the refresh fails? Does the page crash?

Reply:
> Can we increase this to 10 seconds? 5 seems aggressive.

Sure!

Reply:
> In general, what happens if the refresh fails? Does the page crash?

Let me test.

Reply:
https://user-images.githubusercontent.com/17039389/143964115-8fdd77c0-6b67-4ecf-afcd-61cf4c158bdd.mov